src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -113,11 +115,11 @@
   public static final MarcWithHoldingsRequestHelper INSTANCE = new MarcWithHoldingsRequestHelper();
   private final Vertx vertx;
 
-  public static final int POLLING_TIME_INTERVAL = 500;
-
-  public static final int MAX_WAIT_UNTIL_TIMEOUT = 1000*60*20;
+  private static final int REQUEST_ATTEMPTS = 50;
```

Comment:
Can you explain why these were changed to private?

Reply:
It is simple class fields, they should be private. My guess here is that these fields were created among the code and then they were moved to constant by IDEA hotkeys with selected default access as public.